napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -57,6 +98,36 @@ def __init__(self, layer: Layer) -> None:
 
         self.setLayout(LayerFormLayout(self))
 
+        # Buttons
+        self.button_group = QButtonGroup(self)
+        self.panzoom_button = self._radio_button(
+            layer,
+            'pan_zoom',
+            self.MODE.PAN_ZOOM,
+            False,
+            self.PAN_ZOOM_ACTION_NAME,
+            extra_tooltip_text=trans._('(or hold Space)'),
+            checked=True,
+        )
+        self.transform_button = self._radio_button(
+            layer,
+            'transform',
+            self.MODE.TRANSFORM,
+            True,
+            self.TRANSFORM_ACTION_NAME,
+            extra_tooltip_text=trans._('(use Alt-Left mouse click to reset)'),
```

Comment:
Does pre-pending a new line char here work to make the total tooltip 2 lines?

Reply:
Yep, doing something like:

```suggestion
            extra_tooltip_text=trans._('\n(use Alt-Left mouse click to reset)'),
```

makes things look like this:

![tooltip](https://github.com/napari/napari/assets/16781833/0b54f379-fac7-4752-b91d-cae743d47807)

Reply:
yes! with the new line, maybe we don't need the parens and the `use`?
so just
`Alt-Left mouse click to reset`
I think that's still clear, but a bit more concise.

Reply:
Just in case, a preview:

![imagen](https://github.com/napari/napari/assets/16781833/472d6569-8436-44e3-a790-d5f779347434)

Reply:
Nice! Now wondering if we should default to adding a newline for everything before the `extra_tooltip_text`...

Reply:
That makes sense! From a quick check seems like the only other button that has some extra text is pan/zoom:

![imagen](https://github.com/napari/napari/assets/16781833/be524560-9ccb-4565-896e-0e70e2e56e8c)

Maybe something like `Temporarily re-enable by holding Space` could be used as text in case the extra text is added always in a new line?

![imagen](https://github.com/napari/napari/assets/16781833/0bf88973-1fef-4560-a55d-c1b4e4a7ef7f)

Also, should an issue be made to tackle that later or maybe is something worthy to be done here?

Reply:
I mean I feel like the new line would be safe as a default...
but I don't think I would change it in this PR--if at all. It's easy to add the new line, harder to remove it if someone does want a one-liner.

Reply:
Created #6907 to further check the new line behavior related to binding actions to buttons. For the moment, I think things here can be left as they are. Another option could be to change the pan/zoom extra text to include a new line so something like:

```python
self.panzoom_button = self._radio_button(
            layer,
            'pan_zoom',
            self.MODE.PAN_ZOOM,
            False,
            self.PAN_ZOOM_ACTION_NAME,
            extra_tooltip_text=trans._('\n(or hold Space)'),
            checked=True,
        )
```

![imagen](https://github.com/napari/napari/assets/16781833/83f6e98b-3e43-4e28-86ea-e33e433fc6ed)

Or using the text suggestion above:

```python
self.panzoom_button = self._radio_button(
            layer,
            'pan_zoom',
            self.MODE.PAN_ZOOM,
            False,
            self.PAN_ZOOM_ACTION_NAME,
            extra_tooltip_text=trans._('\nTemporarily re-enable by holding Space'),
            checked=True,
        )
```
![329304872-0bf88973-1fef-4560-a55d-c1b4e4a7ef7f](https://github.com/napari/napari/assets/16781833/d64a8849-08d3-401b-a6d3-476ffdc4b49c)